lib/preface_make/bounded_meet_semilattice.ml

```diff
@@ -0,0 +1,23 @@
+module Core (Req : Preface_specs.Bounded_meet_semilattice.WITH_MEET) = Req
+
+module Infix (Core : Preface_specs.Bounded_meet_semilattice.CORE) = struct
+  module Meet_Infix = Meet_semilattice.Infix (Core)
+  include Meet_Infix
```

Comment:
You can use the shortcut `include Meet_semilattice.Infix (Core)` or even simpler `module Infix = Meet_semilattice.Infix`